src/core/src/sim/md/controls.rs

```diff
@@ -104,9 +105,27 @@ impl Control for BerendsenThermostat {
 impl Thermostat for BerendsenThermostat {}
 
 /******************************************************************************/
+
+impl<T> Control for Alternator<T> where T: Control {
+
+    fn control(&mut self, system: &mut System) {
+        if self.can_run() {
+            self.base_mut().control(system)
+        }
+    }
+}
+
 /// Remove global translation from the system
 pub struct RemoveTranslation;
 
+impl RemoveTranslation {
+    /// Create a new `RemoveTranslation"` control, that will
+    /// run every `every` step of the simulation.
+    pub fn new() -> RemoveTranslation {
```

Comment:
For the backward compatibly point of view, nothing prevent anyone from writing `let a = RemoveTranslation;`. We could have a struct with a private empty field (`struct RemoveTranslation(());`), but I am not sure this would be a win. Do we have any example of unit structs and backward compatibility best practises?